techniques/failures/F23.html

```diff
@@ -1,40 +1,60 @@
-<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Failure of  1.4.2 due to playing a sound longer than 3 seconds where
-                    there is no mechanism to turn it off</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link></head><body><h1>Failure of  1.4.2 due to playing a sound longer than 3 seconds where
-                    there is no mechanism to turn it off</h1><section class="meta"><p class="id">ID: F23</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+   <title>Failure of  1.4.2 due to playing a sound longer than 3 seconds where there is no mechanism to turn it off</title>
+   <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link>
+</head>
+<body>
+   <h1>Failure of 1.4.2 due to playing a sound longer than 3 seconds where there is no mechanism to turn it off</h1>
+   <section class="meta">
+      <p class="id">ID: F23</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p>
+   </section>
+   <section id="applicability">
+      <h2>When to Use</h2>
       <p> Applies to all technologies except those for voice interaction. </p>
-   </section><section id="description"><h2>Description</h2>
-      <p> This describes a failure condition for Success Criteria involving sound. If sound does not
-                        turn off automatically within 3 seconds and there is no way to turn the
-                        sound off then Success Criterion 1.4.2 would not be met. Sounds that play over 3 seconds
-                        when there is no mechanism to turn off the sound included in the content
-                        would fall within this failure condition. </p>
-   </section><section id="examples"><h2>Examples</h2>
+   </section>
+   <section id="description">
+      <h2>Description</h2>
```

Comment:
I think it's important to mention that most browsers use heuristics to mute automatically playing audio / video now. That's probably not sufficient as a mechanism, at least that's what ACT concluded when we looked at it. Similarly, several browser support muting individual tabs, we didn't think that qualifies as a mechanism either. I think its important to make that clear.